README.rdoc

```diff
@@ -67,6 +67,28 @@ To install:
 
     bundle exec rake install
 
+=== JRuby
+
+First, you need to download Apache Commons Math 3.6.1 (the JAR, which
+you can find in the binary package). For example, in the NMatrix
+directory, do:
+
+    wget https://www.apache.org/dist/commons/math/binaries/commons-math3-3.6.1-bin.tar.gz
+    tar zxvf commons-math3-3.6.1-bin.tar.gz
+    mkdir ext/nmatrix_java/vendor/
+    cp commons-math3-3.6.1/commons-math3-3.6.1.jar ext/nmatrix_java/vendor/
+
```

Comment:
There is some error with json-version. I will correct it. I left it earlier, because there was no error with travis tests. So, I thought that error was limited to my machine only.